css/auth.css

```
@@ -0,0 +1,185 @@
+.main-btn,
+button {
+  cursor: pointer;
+}
+
+.main-btn:hover,
+button:hover {
+  background: var(--blue200);
+}
+
+.main-btn:active,
+button:active {
+  background: var(--blue300);
+}
+
+.main-btn:disabled,
+button:disabled {
+  background: var(--inactive);
+}
+
+main {
+  height: 61.8rem;
+  margin: 15rem auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+}
+
+.logo-box {
+  width: 39.6rem;
+  height: 13.2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+}
+
+.logo {
+  width: 10rem;
+  height: 10rem;
+}
+
+.pandamarket {
+  width: 26.6rem;
+  height: 12rem;
+  color: var(--blue100);
+}
+
+form {
+  margin-top: 4rem;
+  width: 64rem;
+  height: 44.6em;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.login-input {
+  margin-bottom: 2.4rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.password-container {
+  position: relative;
+}
+
+.password-container input {
+  width: 100%; /* input이 가득 차도록 */
+  padding-right: 4rem; /* 아이콘과 텍스트 간 간격 확보 */
+  box-sizing: border-box; /* 패딩이 width에 포함되도록 설정 */
+}
+
+.eye-icon {
+  position: absolute;
+  right: 1.5rem;
+  top: 42%;
+  transform: translateY(-50%);
+  cursor: pointer;
+  width: 24px;
+  height: 24px;
+}
+
+label {
+  margin-bottom: 1rem;
+  font-size: 1.8rem;
+  font-weight: 700;
+}
+
+input {
+  font-size: 1.6rem;
+  height: 5.6rem;
+  padding: 2rem;
+  border-radius: 1.2rem;
+  background-color: var(--gray100);
+}
+
+input::placeholder {
+  color: var(--inactive);
+}
+
+.btn {
+  height: 5.6rem;
+  color: var(--gray100);
+  font-size: 2rem;
+  font-weight: 600;
+  padding: 1.6rem 12.4rem;
+  border-radius: 4rem;
+  background-color: var(--inactive);
+}
+
+.easy-login {
+  width: 64rem;
```

❗️ 수정요청
해당 태그의 경우 디자인대로 처리가 안 된 것 같아요!
아래처럼 width를 반응형 단위로 해 부모가 줄어들면 같이 줄어들게 해주고, max-width를 통해 제한을 해주면 디자인대로 잘 동작할 것 같아요.

```suggestion
.easy-login {
  max-width: 64rem;
  width: 100%;
```